polaris-react/src/components/Avatar/Avatar.scss

```diff
@@ -19,19 +19,8 @@
   color: var(--p-color-avatar-color-experimental);
   user-select: none;
 
-  #{$se23} & {
-    // stylelint-disable -- se23 overrides
-    --pc-avatar-extra-small-size: 20px;
-    --pc-avatar-small-size: 24px;
-    --pc-avatar-medium-size: 28px;
-    --pc-avatar-large-size: 32px;
-    // stylelint-enable
-    background: var(--p-color-avatar-background-experimental);
-    color: var(--p-color-avatar-color-experimental);
```

Comment:
These changes are already in the `next` branch from a previous pr to consolidate se23 logic.